src/sentry/static/sentry/app/components/hovercard.jsx

```diff
@@ -15,6 +15,7 @@ class Hovercard extends React.Component {
     containerClassName: PropTypes.string,
     header: PropTypes.node,
     body: PropTypes.node,
+    bodyClassName: PropTypes.string,
```

Comment:
If you want to get this in asap this can be a separate PR (for the hovercard stuff)